git_pw/patch.py

```diff
@@ -362,6 +382,14 @@ def list_cmd(
 
     params = []
 
+    if since:
+        LOG.debug('Showing patches since %s', since)
```

Comment:
Can you drop these logs. They don't seem necessary: the user will know what they passed in.